{
	"documenttitle": {
		"text": "Red Hat Security Advisory: python-pillow security update",
		"lang": "en"
	},
	"DocumentType": "Security Advisory",
	"documentpublisher": {
		"type": "Vendor",
		"ContactDetails": "secalert@redhat.com",
		"IssuingAuthority": "Red Hat Product Security"
	},
	"documenttracking": {
		"identification": {
			"ID": "RHSA-2020:3299"
		},
		"Status": "Final",
		"Version": "1",
		"revisionhistory": {
			"revision": {
				"Number": "1",
				"Date": "2020-08-04T07:40:00Z",
				"Description": "Current version"
			}
		},
		"InitialReleaseDate": "2020-08-04T07:40:00Z",
		"CurrentReleaseDate": "2020-08-04T07:40:00Z",
		"generator": {
			"Engine": "Red Hat SDEngine 3.0.0",
			"Date": "2021-08-23T04:47:00Z"
		}
	},
	"documentnotes": {
		"note": [
			{
				"text": "\nAn update for python-pillow is now available for Red Hat Enterprise Linux 8.1 Extended Update Support.\n\nRed Hat Product Security has rated this update as having a security impact of Important. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.\n    ",
				"title": "Topic",
				"type": "Summary",
				"ordinal": "1",
				"lang": "en"
			},
			{
				"text": "\nThe python-pillow packages contain a Python image processing library that provides extensive file format support, an efficient internal representation, and powerful image-processing capabilities.\n\nSecurity Fix(es):\n\n* python-pillow: out-of-bounds reads/writes in the parsing of SGI image files in expandrow/expandrow2 (CVE-2020-11538)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n    ",
				"title": "Details",
				"type": "General",
				"ordinal": "2",
				"lang": "en"
			},
			{
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use",
				"type": "Legal Disclaimer",
				"ordinal": "3",
				"lang": "en"
			}
		]
	},
	"documentdistribution": {
		"text": "Copyright Â© 2021 Red Hat, Inc. All rights reserved.",
		"lang": "en"
	},
	"aggregateseverity": {
		"text": "Important",
		"namespace": "https://access.redhat.com/security/updates/classification/"
	},
	"documentreferences": {
		"reference": [
			{
				"type": "Self",
				"URL": "https://access.redhat.com/errata/RHSA-2020:3299",
				"Description": "https://access.redhat.com/errata/RHSA-2020:3299"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/security/updates/classification/#important",
				"Description": "https://access.redhat.com/security/updates/classification/#important"
			}
		]
	},
	"vulnerability": [
		{
			"ordinal": "1",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "python-pillow: out-of-bounds reads/writes in the parsing of SGI image files in expandrow/expandrow2",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-07-01T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-11538",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2020:3299"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-11538",
						"Description": "CVE-2020-11538"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1852814",
						"Description": "bz#1852814: CVE-2020-11538 python-pillow: out-of-bounds reads/writes in the parsing of SGI image files in expandrow/expandrow2"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Important"
				}
			},
			"DiscoveryDate": "2020-07-01T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.1.0.Z.EUS:python-pillow-0:5.1.1-11.el8_1"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		}
	],
	"producttree": {
		"xmlns": "http://www.icasi.org/CVRF/schema/prod/1.1",
		"branch": [
			{
				"type": "Product Family",
				"name": "Red Hat Enterprise Linux",
				"branch": [
					{
						"type": "Product Name",
						"name": "Red Hat Enterprise Linux AppStream EUS (v. 8.1)",
						"fullproductname": {
							"text": "Red Hat Enterprise Linux AppStream EUS (v. 8.1)",
							"productid": "AppStream-8.1.0.Z.EUS",
							"cpe": "cpe:/a:redhat:rhel_eus:8.1::appstream"
						}
					}
				],
				"fullproductname": {}
			},
			{
				"type": "Product Version",
				"name": "python-pillow-0:5.1.1-11.el8_1",
				"fullproductname": {
					"text": "python-pillow-5.1.1-11.el8_1.src.rpm",
					"productid": "python-pillow-0:5.1.1-11.el8_1",
					"cpe": "cpe:/a:redhat:rhel_eus:8.1::appstream"
				}
			}
		],
		"relationship": [
			{
				"productreference": "python-pillow-0:5.1.1-11.el8_1",
				"relationtype": "Default Component Of",
				"relatestoproductreference": "AppStream-8.1.0.Z.EUS",
				"fullproductname": {
					"text": "python-pillow-0:5.1.1-11.el8_1 as a component of Red Hat Enterprise Linux AppStream EUS (v. 8.1)",
					"productid": "AppStream-8.1.0.Z.EUS:python-pillow-0:5.1.1-11.el8_1",
					"cpe": "cpe:/a:redhat:rhel_eus:8.1::appstream"
				}
			}
		]
	}
}
